src/Microsoft.Extensions.Configuration.AzureAppConfiguration/EmptyTokenCredential.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Azure.Core;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    /// <summary>
+    /// A token credential that provides an empty token.
+    /// </summary>
+    public class EmptyTokenCredential : TokenCredential
```

Comment:
internal, no need to expose this implementation.

Reply:
done